tests/neg/static-no-companion.scala

```diff
@@ -0,0 +1,4 @@
+import annotation.static
+object T {
+  @static val foo = 10 // error: needs companion class
+}
```

Comment:
In dotty, our neg-tests test for presence of errors on lines containing `// error: `. 
We don't check the error message.